Omero (#676)
elixir-europe/rdmkit: pages/tool_assembly/omero_assembly.md

---
title: OMERO
contributors: [Jean-Marie Burel]
tags: [data organisation, storage, data analysis]
page_tag: ome

> summary: Tool assembly for managing imaging data

Reviewer: ```suggestion
summary: Tool assembly for managing microscopy imaging data
```

Author: OMERO supports many imaging data including jpeg, png. We have also worked with marine biologists taking for example images of seabeds or bird colonies. It is not limited to microscopy data, even if our initial focus was on microscope images.

Note that OMERO supports clinical data too.

Reviewer: Valid point. Let's keep it like this then.

---

## What is OMERO?

OMERO is a software platform for managing, sharing and analysing images data. OMERO supports over proprietary 150 file formats, including all major microscopy formats, medical images, digital pathology images, high content screening, etc. using [Bio-Formats](https://www.openmicroscopy.org/bio-formats/). Bio-Formats is a Java software tool for reading proprietary image data and metadata and writing image data using standardized open formats.

> OMERO handles all your images in a secure central repository. Users can **view**, **organize**, **analyze** and **share** data from anywhere via the internet. Users can work with image data and metadata from a Desktop application, from the Web or from 3rd party tools e.g. [Fiji](https://fiji.sc/).

Reviewer: ```suggestion
OMERO handles all your images in a secure central repository. Users can view, [organize](data_organisation), [analyze](analysing) and [share](sharing) data from anywhere via the internet. Users can work with image data and metadata from a Desktop application, from the Web or from 3rd party tools e.g. [Fiji](https://fiji.sc/).
```

OMERO stores image metadata in a relational database and offers a more flexible structure based on HDF5 to store for example, analytical results. This allows analytical results generated by 3rd party softwares e.g. [CellProfiler](https://cellprofiler.org/), [ilastik](https://www.ilastik.org/), etc. to be stored alongside the images.

{% include image.html file="ome_informatics.png" caption="Schematic overview of the OMERO tool assembly." alt="Schematic overview of the OMERO tool assembly." %}

## Who is OMERO intended for?

> OMERO is designed to be an institutional repository. It offers a secure central way for scientists, researchers and data stewards to handle their imaging data. All the image data from a facility can be securely stored and managed, using group permissions and user roles to allow controlled access tailored to your institution. From private repositories for sensitive data to hosting public data for your website and latest publications, the permissions model is designed to meet the range of researchers’ needs. OMERO is tried and tested in hundreds of institutions world-wide, with extensive installation and configuration documentation for system administrators and community support via dedicated mailing lists and forums.

Reviewer: ```suggestion
OMERO is designed to be an institutional repository. It offers a secure central way for scientists, researchers and data stewards to handle their imaging data. All the image data from a facility can be securely stored and managed, using group permissions and user roles to allow controlled access tailored to your institution. From private repositories for [sensitive data](sensitive_data) to hosting public data for your website and latest publications, the permissions model is designed to meet the range of researchers’ needs. OMERO is tried and tested in hundreds of institutions world-wide, with extensive installation and configuration documentation for system administrators and community support via dedicated mailing lists and forums.
```

OMERO is an open-source system designed to be extended and integrated with other tools. The OMERO API allows clients to be written in Java, Python, C++ or MATLAB. The OMERO platform includes a Java client OMERO.insight, a Python-based web client OMERO.web, a Command Line Interface which uses Python, and a Java ImageJ plugin. OMERO also supports a scripting service which allows Python scripts to be run on the server and called from any of the other clients. A vast amount of documentation and code examples for scientists, developers and system administrators are available.

A demo server maintained by the OME team is also available for users wishing to try out, [request an account](https://www.openmicroscopy.org/explore/).

{% include image.html file="omero_integration.png" caption="3rd party tools and OMERO" alt="3rd party tools and OMERO." %}

## Which tasks can be solved with OMERO?

> OMERO can be used for the day-to-day data management of data. Users can remotely view, annotate, analyze, generate figures ready for publication, etc.

Reviewer: ```suggestion
OMERO can be used for the day-to-day data management of data. Users can remotely view, [handle metadata](metadata_management), [analyze](data_analysis), generate figures ready for publication, etc.
```
